src/components/Common/Hr.js

```diff
@@ -4,15 +4,16 @@ import is from 'styled-is'
 
 const Hr = styled.hr`
   height: ${remcalc(1)};
-  margin: ${remcalc(17)} 0 ${remcalc(18)} 0;
+  padding: ${remcalc(17)} 0 ${remcalc(18)} 0;
   border: none;
-  background-color: ${props => props.theme.colors.grey};
+  margin: 0;
+  border-top: 1px solid ${props => props.theme.colors.grey};
   ${is('short')`
     width: 60px;
   `};
   ${is('muted')`
     opacity: 0.25;
-    background-color: ${props => props.theme.colors.white};
+    border-color: ${props => props.theme.colors.white};
```

Comment:
why? Just wondering why the change?

Reply:
because of what i said above? 

> with margin it was being hard to use this component near sections with a different color (it pushed the section down and a 'gap' between the 2 sections appeared. This solutions fixes it and allows for the same usage.

Reply:
oh! Yeah! Changing to padding means you cant use the backgroud color anymore, right